proto/v1alpha1/grafeas.proto

```diff
@@ -1406,6 +1386,26 @@ service Grafeas {
       get: "/v1alpha1/{name=projects/*/notes/*}/occurrences"
     };
   }
+
```

Comment:
I reordered the createoperation methods here so it would generate a less weird grafeas.swagger.json file. It's still technically correct, but would generate a somewhat weird client.

I think this tradeoff is worth it since it only impacts v1alpha1.